src/store/global/maintenance-store/saga.ts

```diff
@@ -2,11 +2,11 @@ import axios from 'axios';
 import { call, put, takeLatest } from 'redux-saga/effects';
 import delay from '@redux-saga/delay-p';
 import { actions } from './slice';
-import { backendUrl, currentChainId } from 'utils/classifiers';
+import { maintenanceUrl, currentChainId } from 'utils/classifiers';
 
 function fetchState() {
   return axios
-    .get(backendUrl[currentChainId] + '/maintenance')
+    .get(maintenanceUrl[currentChainId] + '/maintenance')
```

Comment:
I am just wondering, will there be more endpoints in the maintenance microservice? Because if so, we could have the whole URL in the `maintenanceUrl`.

Reply:
i think in the frontend we can replace it with one url